File: Data/Entities/Log.cs
using System;
using System.Collections.Generic;

namespace FitnessTracker.Data.Entities
{
    public class Log
    {
        public int LogId { get; set; }

        public int UserId { get; set; }

        public User User { get; set; }

        public int Set { get; set; }

        public string Comments { get; set; }

        public DateTime Created { get; set; }

        public DateTime Modified { get; set; }

        public ICollection<LogExercise> LogExercises { get; set; }
    }
}

File: fitness-tracker/src/app/logs/log.ts
export interface IUser{
  id
  username:string
  firstName:string
  lastName:string
  street:string
  city:string
  zip:string
  phoneNumber:string
  email:string
}

export interface ILogExercise{
  exerciseId
  exerciseName:string
  reps:number
  weight:number
  targetRep:number
}

export interface ILog{
  logId:number;
  user:IUser;
  set:number;  
  comments:string;  
  created:string; 
  logExercises:ILogExercise[]
}

export interface ISingleLog{
  logId:number;
  user:IUser;
  set:number;  
  comments:string;  
  created:string; 
  exerciseId
  exerciseName:string
  reps:number
  weight:number
  targetRep:number
}

export interface ISaveLog{
  logId:number;
  user:IUser;
  set:number;  
  comments:string;  
  created:string; 
  logExercise:ILogExercise;
}